lib/bolt/pal/yaml_plan/step/eval.rb

```diff
@@ -5,28 +5,28 @@ class PAL
     class YamlPlan
       class Step
         class Eval < Step
-          def self.allowed_keys
-            super + Set['eval']
-          end
-
           def self.required_keys
-            Set.new
+            Set['eval']
           end
 
-          def initialize(step_body)
-            super
-            @eval = step_body['eval']
+          # Evaluates the step
+          #
+          def evaluate(scope, evaluator)
+            evaluated = evaluator.evaluate_code_blocks(scope, body)
+            evaluated['eval']
           end
 
+          # Transpiles the step into the plan language
+          #
           def transpile
```

Comment:
This is a funny case, but if this is `null` it's valid in the YAML plan but stacktraces when transpiled:
```
...
steps:
  - eval: null
...
```
...
/home/lucy/githubs/bolt/lib/bolt/pal/yaml_plan/step/eval.rb:33:in `transpile': no implicit conversion of nil into String (TypeError)
```

Reply:
Fixed.